geopandas/tests/test_geom_methods.py

```diff
@@ -672,6 +679,29 @@ def test_centroid_crs_warn(self):
         with pytest.warns(UserWarning, match="Geometry is in a geographic CRS"):
             self.g4.centroid
 
+    @pytest.mark.skipif(
+        not compat.SHAPELY_GE_18,
+        reason="ignore_index keyword introduced in shapely 1.8.0",
+    )
+    def test_make_valid(self):
+        polygon1 = Polygon([(0, 0), (0, 2), (1, 1), (2, 2), (2, 0), (1, 1), (0, 0)])
+        polygon2 = Polygon([(0, 2), (0, 1), (2, 0), (0, 0), (0, 2)])
+        linestring = LineString([(0, 0), (1, 1), (1, 0)])
+        testGeoseries = GeoSeries([polygon1, polygon2, linestring])
+        out_polygon1 = MultiPolygon(
+            [
+                Polygon([(1, 1), (0, 0), (0, 2), (1, 1)]),
+                Polygon([(2, 0), (1, 1), (2, 2), (2, 0)]),
+            ]
+        )
+        out_polygon2 = GeometryCollection(
+            [Polygon([(2, 0), (0, 0), (0, 1), (2, 0)]), LineString([(0, 2), (0, 1)])]
+        )
+        output = GeoSeries([out_polygon1, out_polygon2, linestring])
+        testGeoseries.is_valid
```

Comment:
```suggestion
        assert not testGeoseries.is_valid.all()
```
I meant this, to make the assertion checking that before, there is at least one invalid geometry and after there is none.